convert2rhel/systeminfo.py

```diff
@@ -132,51 +141,51 @@ def get_system_release_file_content():
 
         return redhatrelease.get_system_release_content()
 
-    def _get_system_name(self, system_release_content=None):
-        content = self.system_release_file_content if not system_release_content else system_release_content
-        name = re.search(r"(.+?)\s?(?:release\s?)?\d", content).group(1)
-        return name
+    def parse_system_release_content(self, system_release_content=None):
+        """Parse the content of the system release string
 
-    def _get_system_version(self, system_release_content=None):
-        """Return a namedtuple with major and minor elements, both of an int type.
+        If system_release_content is not provided we use the content from the self.system_release_file_content.
 
-        Examples:
-        Oracle Linux Server release 7.8
-        CentOS Linux release 7.6.1810 (Core)
-        CentOS Linux release 8.1.1911 (Core)
+        :param system_release_content: The contents of the system_release file if needed.
```

Comment:
```suggestion
        :keyword system_release_content: The contents of the system_release file if needed.
```

Reply:
To me it seems like `param` is the correct here